### PR TITLE
Curating Owners: pkg/kubelet

### PR DESCRIPTION
**Pull request by apelisse**
cc @euank @vishh @dchen1107 @feiskyer @yujuhong @yifan-gu @derekwaynecarr @saad-ali

In an effort to expand the existing pool of reviewers and establish a
two-tiered review process (first someone lgtms and then someone
experienced in the project approves), we are adding new reviewers to
existing owners files.

If You Care About the Process:
------------------------------

We did this by algorithmically figuring out who’s contributed code to
the project and in what directories.  Unfortunately, that doesn’t work
well: people that have made mechanical code changes (e.g change the
copyright header across all directories) end up as reviewers in lots of
places.

Instead of using pure commit data, we generated an excessively large
list of reviewers and pruned based on all time commit data, recent
commit data and review data (number of PRs commented on).

At this point we have a decent list of reviewers, but it needs one last
pass for fine tuning.

Also, see https://github.com/kubernetes/contrib/issues/1389.

TLDR:
-----

As an owner of a sig/directory and a leader of the project, here’s what
we need from you:

1. Use PR https://github.com/kubernetes/kubernetes/pull/35715 as an example.

2. The pull-request is made editable, please edit the `OWNERS` file to
remove the names of people that shouldn't be reviewing code in the
future in the **reviewers** section. You probably do NOT need to modify
the **approvers** section. Names asre sorted by relevance, using some
secret statistics.

3. Notify me if you want some OWNERS file to be removed.  Being an
approver or reviewer of a parent directory makes you a reviewer/approver
of the subdirectories too, so not all OWNERS files may be necessary.

4. Please use ALIAS if you want to use the same list of people over and
over again (don't hesitate to ask me for help, or use the pull-request
above as an example)

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/36528)

### CODE REVIEW

**Comment by yujuhong** [comment]
@apelisse I find it hard to believe that @Random-Liu is not on the list at all....Perhaps there's a bug in the algorithm that excluded github handle with capital letters (just a random guess)?

**Comment by apelisse** [comment]
Feel free to add names that you find relevant :-)

**Comment by yujuhong** [comment]
> Feel free to add names that you find relevant :-)

I am more curious about the algorithm as I don't know how that was possible.

**Comment by yujuhong** [comment]
> I am more curious about the algorithm as I don't know how that was possible.

Just to provide one more example, @Random-Liu doesn't appear on the "reviewers" list in #36467 as well, even though he is in the OWNER file and has been the top contributor of the package.

**Comment by apelisse** [comment]
@yujuhong @calebamiles Admittedly, there is a bug in the process that leads is name to be NOWHERE...

File: pkg/kubelet/cm/OWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- derekwaynecarr
+- vishh
+- liggitt
+- justinsb
+- ncdc
+- timstclair
+- dims
+- jingxu97
+- david-mcmahon
+- '249043822'
```
Comment (timstclair) [inline pkg/kubelet/cm/OWNERS]: What's this?

Comment (apelisse) [inline pkg/kubelet/cm/OWNERS]: Someone's user name ... :-)

Comment (apelisse) [inline pkg/kubelet/cm/OWNERS]: https://github.com/249043822

Comment (apelisse) [inline pkg/kubelet/cm/OWNERS]: indeed

**Comment by resouer** [comment]
@yujuhong One of the possible reason is current implementation gives "latest" commits much weight (e.g. latest top N committers ), and another bug I can see is it can not distinguish `typo`and fixes like `sed s/xxx/yyy/g`. However, both of them need manual intervention to decides `commits that matters`. I talked with @feiskyer and @dchen1107 in kubecon about this issue, but no better idea. 

For now, I can help to scan the list in this PR based on history commits if you are OK with that.

**Comment by saad-ali** [comment]
`pkg/kubelet/volumemanager` LGTM

File: pkg/kubelet/OWNERS [hunk 2]
```diff
@@ -1,4 +1,45 @@
-assignees:
-  - dchen1107
-  - vishh
-  - yujuhong
+approvers:
```
Comment (resouer) [inline pkg/kubelet/OWNERS]: What's the relationship of this OWNERS file against OWNERS under subdirectory? Do we still need this?

**Comment by yujuhong** [comment]
@resouer, I spoke to @apelisse offline and there was a problem in the code that couldn't handle github handles such as "Random-Liu". That's why he suggested more usernames in the similar situation in https://github.com/kubernetes/kubernetes/pull/36528#issuecomment-259547727 :-) Fortunately, there aren't many user names with this issue, so we can just manually add them if needed.

**Comment by dchen1107** [comment]
Here is my two cents:

1) I will leave sig-node alias as is. It is used for anyone interested in kubernetes node related work, not limited to developer.
2) But I will create a separate sig-node-reviewer alias to serve this purpose. Please note that reviewer is also hierarchical, and we can remove a lot of duplicate name from current lists in this pr. 
3) I would like to start the list smaller.

**Comment by feiskyer** [comment]
> 2) But I will create a separate sig-node-reviewer alias to serve this purpose. Please note that reviewer is also hierarchical, and by using the alias we can remove a lot of duplicate name from current lists in this pr. 

+1. We could add alias to [OWNERS_ALIASES](https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES) and refer it in OWNERS.

**Comment by apelisse** [comment]
@k8s-bot kops aws e2e test this

**Comment by apelisse** [comment]
@timstclair @dchen1107 @resouer I vastly simplified this PR, could you PTAL?

**Comment by dchen1107** [comment]
Nice! LGTM

**Pull request merged**